doc/source/examples/analysis/alignment_and_rms/pairwise_rmsd.ipynb

```diff
@@ -108,12 +117,12 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Step    98/98 [100.0%]\n"
+      "Step   102/102 [100.0%]\n"
```

Comment:
locally this progress bar doesn't show up with the current code, you need to passs `verbose=True` to `run()`. Also as of v1.0 the progress bar should be tdqm not the old progress meter.

Reply:
I'm ok with it not showing up unless you think it's necessary? Re-ran with 1.0 to get rid of the bar.

Reply:
As long as we show off the progress bar somewhere it doesn't matter. That being said, maybe a "unified style" might be favourable, i.e. all progress bars or all no progress bars.